src/test/resources/features/NewCheckingWithLowAmount.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Creating a new checking account with low amount
+
+  Scenario: Create a new standard checking account with low amount
```

Comment:
Since there were some PRs merged to master, you have to pull latest and my advice is to see if there are any steps that you can reuse, thus can remove from your side and update the feature file